poker-pals/src/components/pokerComponents/ReportPocket.vue

```diff
@@ -49,88 +132,76 @@
         width: 13%;
         height: 13%;
         margin: 2% 0% 0% 48%;
-        background: #666666;
         cursor: pointer;
     }
     .buttonXOuter:active{
         transform: translateY(4%);
     }
-    .buttonX{
-        float: left;
-        width: 90%;
-        height: 90%;
+    .close {
+        padding: 0% 3%;
         font-size: 1.5vw;
-        margin: 5% 0% 0% 5%;
-        background: #bf214b;
-        color: white;
     }
-    .mainTitle{
+    .close:hover {
+        background-color: white;
+    }
+    .offending-user-header {
         float: left;
         width: 94%;
         height: 10%;
-        font-size: 1.25vw;
-        margin: 0% 0% 0% 3%;
-        background: transparent;
+        font-size: 1vw;
         text-align: left;
         color: black;
     }
-    .witnessQuestionText{
+    .prompt {
         float: left;
         width: 94%;
         height: 10%;
         font-size: 1vw;
-        margin: 5% 0% 0% 3%;
-        background: transparent;
+        margin: 5% 0% 0% 0%;
         text-align: left;
         color: black;
     }
-    .selector{
-        float: left;
-        width: 70%;
-        height: 10%;
-        font-size: 1vw;
-        border-radius: 2%/12%;
-        margin: 0% 0% 0% 3%;
-    }
-    .describeReasonText{
+    .warning {
         float: left;
         width: 94%;
         height: 10%;
-        font-size: 1vw;
-        margin: 5% 0% 0% 3%;
+        font-size: 0.8vw;
         background: transparent;
         text-align: left;
-        color: black;
+        color: rgb(196, 0, 0);
+        font-style: italic;
     }
-    .inputTextBox{
-        float: left;
-        width: 94%;
-        height: 17%;
-        font-size: 1vw;
-        margin: 0% 0% 0% 3%;
-        text-align: left;
-        resize: none;
+    .hidden {
+        display: none;
     }
-    .buttonCancel{
+    .selector{
```

Comment:
Would `select` work rather than  `.selector`? I was mildly confused as I thought of "css selectors".

Reply:
Renamed the class instead of styling all select elements.  Hope that's okay by you.